src/client/bridge/gateway/shard_messenger.rs

```diff
@@ -107,9 +107,9 @@ impl ShardMessenger {
     /// ```
     ///
     /// [`Event::GuildMembersChunk`]:
-    /// ../../model/event/enum.Event.html#variant.GuildMembersChunk
-    /// [`Guild`]: ../../model/guild/struct.Guild.html
-    /// [`Member`]: ../../model/guild/struct.Member.html
+    /// ../../../model/event/enum.Event.html#variant.GuildMembersChunk
```

Comment:
It doesn't look like we typically break links on multiple lines if it causes the line to be over 80 characters, so let's one-line these.